### ISSUE TITLE
NVDa responds slowly when browsing the web

### ISSUE THREAD

**Issue by HBM301** [ahmedthebest31/navsounds#1]
Hello, thanks for creating a great addon i really like it, but there is a problem: that nvda responds very slow when browsing the web, i tried uninstalling the addon and nvda works as fast as usual please fix it thanks,This is my first time joining github. If there are any mistakes, please forgive me

**Comment by ahmedthebest31** [comment]
Most likely my addon conflicting with any other one you have an installed in your nvda
Because it's working perfectly on my system and already I publish it on my YouTube channel and no one told me about this issue at all everything claims it worked just fine
Can you please tell me your nvda version

 And if you can give me a quick record while you using this addon  to help me identify the issue and fix it ASAP

**Comment by HBM301** [comment]
Hi, I'm not sure if any add-ons conflict with my nvda, but I tried creating a portable version with the default configuration and the situation is still the same. and i am using nvda 2023.2,everything works smoothly but it's only when browsing the web that the problem appear

**Comment by ahmedthebest31** [comment]
Hi there,

Thank you for getting in touch.

I'd like to clarify that this issue doesn't seem to originate from my addon but rather from NVDA itself. The problem arises when NVDA attempts to produce a significant number of sounds, especially on devices with limited performance capabilities.

I conducted tests using this addon on three different devices:

1. My main computer, an Asus system featuring an AMD Ryzen processor.
2. My first laptop, a 10th generation Lenovo laptop with an i3 processor.
3. My HP EliteBook 8520.

During these tests, the addon functioned perfectly without any issues. I've attached a short video demonstrating the addon in action.

Unfortunately, I am unable to fix the addon at this time as the problem appears to be on the NVDA screen reader side. I hope they can resolve this issue in the near future.


https://github.com/ahmedthebest31/navsounds/assets/30302865/10a1096c-5c66-45d8-a0cd-ac90b7d76f1a

Thank you for your understanding.

https://github.com/ahmedthebest31/navsounds/assets/30302865/a31eb4b3-7cc6-43ae-b756-8c3802077d35
